aws-rds-dbinstance/aws-rds-dbinstance.json

```diff
@@ -0,0 +1,431 @@
+      "pattern": "^[a-zA-Z]{1}(?:-?[a-zA-Z0-9]){0,254}$"
+    },
+    "DBSubnetGroupName": {
+      "type": "string",
+      "description": "A DB subnet group to associate with the DB instance. If you update this value, the new subnet group must be a subnet group in a new VPC.",
+      "pattern": "^[a-zA-Z]{1}(?:-?[a-zA-Z0-9]){0,254}$"
+    },
+    "DeleteAutomatedBackups": {
+      "type": "boolean",
+      "description": "A value that indicates whether to remove automated backups immediately after the DB instance is deleted. This parameter isn't case-sensitive. The default is to remove automated backups immediately after the DB instance is deleted."
+    },
+    "DeletionProtection": {
+      "type": "boolean",
+      "description": "A value that indicates whether the DB instance has deletion protection enabled. The database can't be deleted when deletion protection is enabled. By default, deletion protection is disabled."
+    },
+    "Domain": {
+      "type": "string",
+      "description": "The Active Directory directory ID to create the DB instance in. Currently, only MySQL, Microsoft SQL Server, Oracle, and PostgreSQL DB instances can be created in an Active Directory Domain."
+    },
+    "DomainIAMRoleName": {
+      "type": "string",
+      "description": "Specify the name of the IAM role to be used when making API calls to the Directory Service."
+    },
+    "EnableCloudwatchLogsExports": {
+      "type": "array",
+      "uniqueItems": true,
+      "insertionOrder": false,
+      "items": {
+        "type": "string"
+      },
+      "description": "The list of log types that need to be enabled for exporting to CloudWatch Logs. The values in the list depend on the DB engine being used."
+    },
+    "EnableIAMDatabaseAuthentication": {
+      "type": "boolean",
+      "description": "A value that indicates whether to enable mapping of AWS Identity and Access Management (IAM) accounts to database accounts. By default, mapping is disabled."
+    },
+    "EnablePerformanceInsights": {
+      "type": "boolean",
+      "description": "A value that indicates whether to enable Performance Insights for the DB instance."
+    },
+    "Engine": {
+      "type": "string",
+      "enum": [
+        "aurora",
+        "aurora-mysql",
+        "aurora-postgresql",
+        "mariadb",
+        "mysql",
+        "oracle-ee",
+        "oracle-se2",
+        "oracle-se1",
+        "oracle-se",
+        "postgres",
+        "postgresql-license",
+        "sqlserver-ee",
+        "sqlserver-se",
+        "sqlserver-ex",
+        "sqlserver-web"
+      ],
+      "description": "The name of the database engine that you want to use for this DB instance."
+    },
+    "EngineVersion": {
+      "type": "string",
+      "description": "The version number of the database engine to use."
+    },
+    "Iops": {
+      "type": "integer",
+      "minimum": 1000,
```

Comment:
Should we specify a maximum 80000 as it mentioned in doc?
Reference: https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_Storage.html#USER_PIOPS

Reply:
It is 256k for Oracle. I'd say let's leave this out as it is engine-specific and would be rejected by WS if exceeds the limit anyways.